lib/tasks/reporting.rake

```diff
@@ -80,6 +80,47 @@ namespace :reporting do
     end
   end
 
+  desc "A CSV report of the number of publications in draft state by organisation between the dates specified [year-month-day-hour-min]"
+  task :number_of_draft_publications_by_organisation_by_date_range, %i[start_date end_date] => :environment do |_t, args|
+    if args[:start_date].present? && args[:end_date].present?
+      start_date = Time.parse(args[:start_date])
+      end_date = Time.parse(args[:end_date])
+    end
```

Comment:
Should we raise an error here if these arguments are missing? Perhaps like:

```ruby
raise "Missing start_time or end_time" unless args[:start_date].present? && args[:end_date].present?
```

and then we don't need the `if` block.